src/parser.rs

```diff
@@ -153,7 +153,7 @@ impl Amount {
 
         Ok(Amount {
             kind,
-            dollars: Self::mk_string(caps.get(1)).replace(",", ""),
+            dollars: Self::mk_string(caps.get(1)).replace(',', ""),
```

Comment:
Is this change driven by changes to the API? For consistency, I'd prefer the replacement also be a char (`''`) instead of a string (`""`).